packages/office-ui-fabric-react/src/components/Layer/examples/Layer.Customized.Example.tsx

```diff
@@ -1,25 +1,22 @@
-import * as React from 'react';
 import { Customizer } from '@uifabric/utilities';
-import { Panel } from 'office-ui-fabric-react/lib/Panel';
 import { Checkbox } from 'office-ui-fabric-react/lib/Checkbox';
 import { LayerHost } from 'office-ui-fabric-react/lib/Layer';
+import { Panel } from 'office-ui-fabric-react/lib/Panel';
+import * as React from 'react';
 
-export interface ILayerCustomizedExampleState {
+interface IState {
   showPanel: boolean;
   trapPanel: boolean;
 }
 
-export class LayerCustomizedExample extends React.Component<{}, ILayerCustomizedExampleState> {
-  constructor(props: {}) {
-    super(props);
-
-    this.state = {
-      showPanel: false,
-      trapPanel: false
-    };
-  }
```

Comment:
It's not necessary to explicitly define a constructor except for very specific circumstances (like debouncing)